README.md

```diff
@@ -15,22 +15,22 @@ The main features of it are:
 
 ### Architecture
 
-This project consists of 2 main parts: Worker and REST API 
+This project consists of 2 main parts: Observer and REST API 
 
-Worker - service that periodically fetch latest data from **data providers** (like coinmarketcap), parse it to the common data structure, set the parsed data to the cache (Redis)
+Observer - service that periodically fetch latest data from **data providers** (like coinmarketcap), parse it to the common data structure, set the parsed data to the cache (Redis)
 
 REST API - allows to get cached data through REST HTTP API
 
 ### Getting started
 
 1. Spin up a Redis instance: `docker run -it -p 6379:6379 redis`
-1. Start the app: `make start`
+2. Start the app: `make start`
    1. Explore the API: [http://localhost:8423/swagger/index.html](http://localhost:8423/swagger/index.html)
    1. Use the API:
       * `curl -v "http://localhost:8421/v1/market/info?coin=60" | jq .`
       * `curl -v -X POST 'http://localhost:8421/v1/market/ticker' -H 'Content-Type: application/json' -d '{"currency":"ETH","assets":[{"type":"coin","coin":60}]}'`
       * `curl -v "http://localhost:8421/v1/market/charts?coin=60&time_start=1574483028" | jq .`
-1. When done: `make stop`
+3. When done: `make stop`
```

Comment:
nitpick: GitHub's Markdown flavor automatically increases the rendering of numbered lists. So what we had before I argue was better

Or is this also rendered somewhere else?

Reply:
Did not know, but this way steps numeration sequence more explicite, instruction for humans not for machine to read.

Reply:
Don't feel that strong on this but IMO it's better to just use `1.` so you don't need to maintain the order